.github/workflows/convert.yml

```diff
@@ -0,0 +1,71 @@
+name: convert-yaml-to-json
+
+on:
+  push:
+    paths:
+      - "yaml/**.yaml"
+
+jobs:
+  convert-yaml:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+          cache: "pip"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: Run python script
+        run: |
+          if [ -d "yaml" ] && [ "$(ls -A yaml/*.yaml 2>/dev/null)" ]; then
+            python convert-yaml.py yaml json
+          else
+            echo "No YAML files to process."
+          fi
+
+      - name: Upload JSON artifacts for validation
+        uses: actions/upload-artifact@v4
+        with:
+          name: json-files
+          path: json/*.json
+
+  validate-json:
+    needs: convert-yaml
+    uses: ./.github/workflows/validate.yml
+    with:
+      should_download: true
+
+  commit-json:
+    runs-on: ubuntu-latest
+    needs: validate-json
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Check for validation success and file changes
+        run: |
+          if [ "${{ needs.validate-json.outputs.is_valid }}" != "true" ]; then
+            echo "JSON is not valid. Skipping commit."
+            echo "NO_COMMIT=true" >> $GITHUB_ENV
+          elif git diff --quiet; then
+            echo "No changes to commit"
+            echo "NO_COMMIT=true" >> $GITHUB_ENV
+          fi
+
+      - name: Commit changes
+        if: ${{ env.NO_COMMIT != 'true' }}
+        run: |
+          git config --global user.name "github-actions"
+          git config --global user.email "github-actions@github.com"
```

Comment:
We might want to use a branded account here? For our website builds we use https://github.com/thundernest-bot

If you tie the email to a secret we can probably re-use that. Thoughts @Sancus ?